src/macros/macromanager.cpp

```diff
@@ -0,0 +1,47 @@
+#include "macromanager.h"
```

Comment:
Please fix your #include directors by including the relative path as already mentioned repeatedly.